quic/s2n-quic-core/src/packet/interceptor.rs

```diff
@@ -151,6 +158,17 @@ where
     Tx: 'static + Send + havoc::Strategy,
     R: 'static + Send + havoc::Random,
 {
+    #[inline]
+    fn intercept_rx_remote_port(&mut self, _subject: &Subject, port: &mut u16) {
+        let mut new_port = port.to_le_bytes();
+        let mut new_port = EncoderBuffer::new(&mut new_port);
+        self.rx.havoc(&mut self.random, &mut new_port);
```

Comment:
I'm wondering if it would be best to have a different strategy for ports, since we're only modifying 2 bytes. We should also have a strategy that holds a previous value for a number of calls, just so the receiver has time to respond to the migration.

Reply:
I've added a `Hold` strategy and separated out the strategies for `havoc_rx` from `havoc_port`. One thing I'm not sure on though is that we only allow for 4 migrations (5 total paths) before we start just dropping all packets, so pretty quickly the fuzzing of remote port doesn't have much effect unless we close the connection and reopen it.